native/src/__mocks__/react-i18next.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react'
 type I18nType = typeof import('i18next').default
 const realModule = jest.requireActual('react-i18next')
 
-const withTranslation = (namespace: string) => (Component: React.ComponentType<any>): React.ComponentType<any> => {
+const withTranslation = () => (Component: React.ComponentType<any>): React.ComponentType<any> => {
```

Comment:
I would pefere to use `_namespace:string` to make clear that it is unused. That way it is clear that this parameter exist.

How does TS deal with this? Isn't the type of withTranslation changed in that case to `() -> Component` instead of `string -> Component`?